Symfony/CS/Tests/Fixer/Contrib/SingleQuoteFixerTest.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Tests\Fixer\Contrib;
+
+use Symfony\CS\Tests\Fixer\AbstractFixerTestBase;
+
+/**
+ * @author Gregor Harlan <gharlan@web.de>
+ */
+class SingleQuoteFixerTest extends AbstractFixerTestBase
+{
+    /**
+     * @dataProvider provideTestFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->makeTest($expected, $input);
+    }
+
+    public function provideTestFixCases()
+    {
+        return array(
+            array('<?php $a = \'\';', '<?php $a = "";'),
+            array('<?php $a = \'foo bar\';', '<?php $a = "foo bar";'),
+            array('<?php $a = \'foo bar\';'),
+            array('<?php $a = "foo \'bar\'";'),
```

Comment:
agree with @gharlan . quotes case is to use one in most cases, but to not force it for string that would need a lot of escaping.